### DISCUSSION TITLE
TCL-68 Styling ideas 💅

### DISCUSSION THREAD

**Discussion by cafloyd** [the-collab-lab/tcl-68-smart-shopping-list#29] [Ideas]
Let's use this space to keep track of our ideas for the look and feel of our app ahead of working on issue #14 !

**Comment by cafloyd** [comment]
As @luisaugusto pointed out, you can get some inspiration from past TCL shopping list apps! You can browse the deployed apps on the [TCL website](https://the-collab-lab.codes/developers/), or look through [other repos in the TCL org](https://github.com/orgs/the-collab-lab/repositories) on Github

**Comment by DevinaG007** [comment]
Not necessarily a "styling" idea, but just an idea in general - we should make our tagline "Nacho average shopping list!"

**Comment by DevinaG007** [comment]
I really liked some of the lists that had little icons next to the items, and I think it would be super cute to add an animation or wow feature to our list. I had a thought for something fun we could do. It would challenge our skills with animations a bit and we would need to add an [emoji library](https://www.cssscript.com/emoji-picker-button/), but I think it would be super cute.

In order to implement this, we would need to add a feature to our Add Item component where a user can also choose an emoji to be displayed with the item in the list. I was thinking that we could add a little animation so that when the user clicks on an item, the icon they've selected will jump into the shopping bag. I made a little sketch to show what I mean, but we don't need to keep any of the other stylistic elements that I made in this picture. What do you guys think? This is only one idea, so if you guys have other ideas, I'm open to whatever! 

![smart-shopping-list-idea](https://github.com/the-collab-lab/tcl-68-smart-shopping-list/assets/137969744/1a17d8d1-0ae2-4f7c-b492-ba319dd89e49)

**Reply by DevinaG007** [reply]
Also, if the emoji library is too complicated of an idea, we could always just use one generic image of an item. I thought it would be fun to let the user pick their own though! Perhaps we could have a default image if the user elects not to choose an emoji or something.

**Reply by andiedoescode** [reply]
I really like the idea of the being able to choose an emoji + a default image!

**Reply by cafloyd** [reply]
Love this idea!! Depending on the amount of capacity you all have, this idea could also easily be broken down into and MVP version and a post-MVP version.

(For example, MVP could be just having a generic image, stretch goal could be customizing the images, post-MVP could be adding animation)

**Reply by sdmisra** [reply]
I think this is a fantastic idea - I haven't used emoji libraries at all yet, and would love to see if we can add customizable elements for the lists.

**Comment by andiedoescode** [comment]
The option to have dark mode could be a nice UX addition. (I live my life in dark mode where available!)

**Comment by andiedoescode** [comment]
Perhaps not a styling option and moved to be an issue: Having some sort of indication about which list has been selected to display across all our views (Manage List and List especially)

**Comment by DevinaG007** [comment]
Moodboard: https://www.canva.com/design/DAF_xxQqbvM/3w5i8D6Xwhrx4F1Ua3cy5g/edit?utm_content=DAF_xxQqbvM&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by DevinaG007** [comment]
Discussion Points for Meeting:

- Tone Words
- Moodboard
- Font
- Vanilla CSS or Framework?
- Colors
- Building a wireframe
